test/test_ops.py

```diff
@@ -84,14 +84,19 @@ def variant_out_fn(*args, **kwargs):
             else:
                 variant_out_fn = variant
             partial_fn = partial(variant_out_fn, **sample.kwargs)
```

Comment:
Since partial_fn lives entirely within `fn` now I think its definition can be moved inside it, too